sdcm/utils/cloud_monitor/resources/instances.py

```diff
@@ -96,6 +100,32 @@ def get_keep_alive_gce_instance(instance):
         return ""
 
 
+class AzureInstance(CloudInstance):
+    pricing = AzurePricing()
+
+    def __init__(self, instance: VirtualMachine, resource_group: str):
+        tags = instance.tags or {}
+        super().__init__(
+            cloud="azure",
+            name=instance.name,
+            instance_id=resource_group,
+            region_az=instance.location,
+            state="running",
+            lifecycle=InstanceLifecycle.SPOT if instance.priority == "Spot" else InstanceLifecycle.ON_DEMAND,
+            instance_type=instance.hardware_profile.vm_size,
+            owner=tags.get("RunByUser", NA),
+            # azure is not providing vm creation time - for machines that don't have creation_time, set default in the past
```

Comment:
shouldn't we add the creation time as a label of the instance in this case?

Reply:
I think we are doing that, but this is a fallback here, for cases it's an instance wasn't create by SCT (or lost it's tagging for any reason)

I would rather add the needed code to support, create_time=None, and not spoofing a date like that, it would only cause confusion.

Reply:
yes, this is fallback for resources not created by SCT.

I'll add code to support create_time=None

Reply:
ok, added - see separate commit for this.